lib/cn/cn.c

```diff
@@ -497,7 +508,17 @@ cn_mblocks_destroy(
         if (kcompact)
             continue;
         cn_delete_blks(ds, &list[lx].vblks, n_committed);
+
+        /* Same logic as cn_delete_blks() applied to one block. Candidate for
+        * refactor.
+        */
+        if (n_committed > 0) {
+            delete_mblock(ds, &list->hblk);
+        } else {
+            abort_mblock(ds, &list->hblk);
+        }
     }
+
```

Comment:
roundup_size() below... why does it exist?  Why not use roundup() from param.h?

Reply:
Fixing in my PR